app/src/androidTest/java/org/simple/clinic/util/RelativeTimestampGeneratorAndroidTest.kt

```diff
@@ -26,21 +29,22 @@ class RelativeTimestampGeneratorAndroidTest {
 
   @Test
   fun it_should_generate_timestamp_strings() {
+    val generator = RelativeTimestampGenerator()
     val todayDateTime = LocalDateTime.of(2018, 7, 29, 16, 58)
 
     val todayTime = LocalDateTime.of(2018, 7, 29, 2, 0, 0).atOffset(ZoneOffset.UTC).toInstant()
-    assertThat(generator.generate(todayDateTime, todayTime).displayText(context)).isNotEmpty()
+    assertThat(generator.generate(todayDateTime, todayTime).displayText(context, formatter)).isNotEmpty()
```

Comment:
`.isNotEmpty()` does not tell the whole story here. There is no means for the maintainer to know what the expectations are, and what is/was the behavior of these calls.

We need to assert them against actual string values, which I understand is coupled to the Android UI. Without those, these tests don't help a lot.

Reply:
That is true. The reason these tests were added initially was as a failing test to fix a bug which caused a crash when formatting an `Instant` (because an `Instant` by itself is not sufficient to format a local date, you also need a time zone). 

If we do check the string here, this test will be deeply coupled to the UI and will be brittle. I feel we can remove this test.

Reply:
If this can be removed, then please remove it. Brittle tests are better than tests that don't add any value. I fail to see how these tests provide any value at the moment.